pandas/core/base.py

```diff
@@ -951,22 +957,36 @@ def max(self):
         >>> idx.max()
         ('b', 2)
         """
+        nv.validate_minmax_axis(axis)
         return nanops.nanmax(self.values)
```

Comment:
Why isn't skipna passed? Is that a followup item?

Reply:
I had considered this a follow-up item, but am now leaning towards fixing these now.  Good catch.

Reply:
This turns out to be more involved than expected, requires some changes in Series._reduce and IndexOpsMixin._reduce.  Neither of which is all that invasive, but changing them entails changing/fixing/testing behavior for other reductions I hadn't planned on doing in this PR.

Currently going through to get that all done here.  I'm also OK with doing this in two phases so as to take things out of #24024 sooner rather than later.